examples/01_minimal/src/link.js

```diff
@@ -36,6 +36,7 @@ const mocks = {
 };
 
 const schema = makeExecutableSchema({ typeDefs });
-addMocksToSchema({ mocks, schema });
 
-export default new SchemaLink({ schema });
+const schemaWithMocks = addMocksToSchema({ schema, mocks });
```

Comment:
So, it no longer change the schema, but returns a new one. I didn't find that. Cool.